packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -19,6 +19,8 @@ import {
   techdocsPlugin,
   TechDocsReaderPage,
 } from '@backstage/plugin-techdocs';
+import { TechDocsAddons } from '@backstage/plugin-techdocs-react/alpha';
```

Comment:
Gotta say I'm not super fond of adding alpha features into the `create-app` template. Would a middle-ground perhaps be to add a comment with a link to docs for how to set it up or something like that instead?

Thoughts on this @backstage/maintainers / @backstage/techdocs-core?

Reply:
FWIW, I'm (we're?) hoping to remove the `alpha` from this before the 1.2 release.

Reply:
@Rugvip, just to let you know that we will wait for the next release to avoid adding an alpha feature 🙂 , thanks for your review!

Reply:
@iamEAP and @Rugvip, version alpha removed!